bot.py

```
@@ -4,6 +4,29 @@
+    "/finish": "Finish adding music to a playlist.",
+    "/myplaylists": "List your playlists.",
+    "/show_playlist": "Show tracks in a playlist.",
+    "/share": "Share a playlist with others.",
+    "/remove_track": "Remove a track from a playlist.",
+    "/remove_playlist": "Delete a playlist.",
+    "/rename": "Rename a playlist.",
+    "/setcover": "Set a cover image for a playlist.",
+    "/help": "Show this help message."
+}
+
+@dp.message(F.text == "/help")
```

Ruff (0.12.2)

Undefined name `dp`

(F821)